lib/vendors/axis-communications/sv_vendor_axis_communications.c

```diff
@@ -818,6 +818,19 @@ get_axis_communications_supplemental_authenticity(void *handle,
   return status;
 }
 
+/**
+ * Retrieves the certificate chain from the vendor handle. */
+const char *
+get_axis_communications_certificate_chain(void *handle)
+{
+  if (!handle) return NULL;
+
+  sv_vendor_axis_communications_t *self = (sv_vendor_axis_communications_t *)handle;
+
+  // Return the certificate chain
```

Comment:
Redundant comment. I think the reader is clever enough to understand that from the line below.